.circleci/config.yml

```diff
@@ -222,6 +222,12 @@ commands:
             echo 'export BASIC_AUTH_USERNAME=$BASIC_AUTH_USERNAME' >> $BASH_ENV
             echo 'export BASIC_AUTH_PASSWORD=$BASIC_AUTH_PASSWORD' >> $BASH_ENV
 
+            echo 'export DCA_DYNAMICS_ORG_URL_DEV=$DCA_DYNAMICS_ORG_URL' >> $BASH_ENV
+            echo 'export DCA_DYNAMICS_CLIENT_ID_DEV=$DCA_DYNAMICS_CLIENT_ID' >> $BASH_ENV
+            echo 'export DCA_DYNAMICS_SECRET_DEV=$DCA_DYNAMICS_SECRET' >> $BASH_ENV
+            echo 'export DCA_DYNAMICS_TENANT_ID_DEV=$DCA_DYNAMICS_TENANT_ID' >> $BASH_ENV
+            echo 'export FEATURE_DYNAMICS_PUBLIC_MOVERS_DEV=$FEATURE_DYNAMICS_PUBLIC_MOVERS' >> $BASH_ENV
+
```

Comment:
[pebble] Even though we don't know what these values will be, should we set these for `set-prod-env-vars`?

Reply:
Just thinking so that when the time comes to officially release this it's just a change in CircleCI.

Reply:
yeah sure, I don't see why not, I will add them!